.changeset/lucky-eels-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/keystone': patch
```

Comment:
Moderately irrelevant here given other PRs will bump this to major anyway, but I'm not keen on changesets for internal refactors.

Given most people understand semver to mean:

* Major = breaking API change (or potentially breaking behavioural change)
* Minor = new feature
* Patch = bug fix

... this is none of those, and not really of interest to anyone consuming the package, so I'd leave it unversioned in this PR and unmentioned in the changelog.

This level of detail is (imo) what the commit history is for, and if this were the only change to land this release cycle I don't think it warrants a new version being published.

Open to disagreement here, that's my hot take, I'll open a separate PR to contributing with a proposal for our guidelines here because it seems to come up a lot 🙂

Reply:
Nerd sniped myself. #4462